packages/lesswrong/server/resolvers/arbitalPageData.ts

```diff
@@ -24,69 +31,125 @@ async function getArbitalPageData(pageAlias = "hyperexistential_separation") {
   return await response.text()
 }
 
-// Make a string safe for use in the text part of a link, by backslash-escaping brackets.
-// (This is not particularly well tested. However, it's a text transform over markdown not
-// over HTML, so even if it's totally wrong it can only break links, not cause XSS.)
+/**
+ * Make a string safe for use in the text part of a link, by backslash-escaping brackets.
+ * (This is not particularly well tested. However, it's a text transform over markdown not
+ * over HTML, so even if it's totally wrong it can only break links, not cause XSS.)
+ */
 function markdownEscapeLinkText(text: string) {
   return text.replace(/[[\]]/g, match=>'\\'+match);
 }
 
+/**
+ * Fetch an Arbital page for link-previewing (by requesting it from the Arbital server).
+ * This should not be used directly; use getArbitalPageWithCache instead.
+ */
+async function fetchArbitalPageAsHtml(pageAlias: string): Promise<ArbitalPageData|null> {
+  const rawRoomData: any = await fetchArbitalPageData(pageAlias)
```

Comment:
room data?  assume this was copied from mozilla hubs at some point in the past

Reply:
Plausible; I assumed Arbital must have a "room" concept that I didn't know about, and moved this around without renaming it, but it makes sense that it might actually have been a copy-paste tracing back to the Mozilla Hubs integration.